### DISCUSSION TITLE
Adding liquidity

### DISCUSSION THREAD

**Discussion by rowlanja** [Jeiwan/uniswapv3-book#94] [Milestone 0. Introduction]
When adding liquidity we provide two ticks, an upper and a lower tick bound. I noticed that the book states we "Add the liquidity to both ticks". How is this adding liquidity to the range between the ticks? it seems like its just adding liquidity to two points along a range

**Comment by vishwa7dcap786** [comment]
Refer this one ---->   https://github.com/Jeiwan/uniswapv3-book/discussions/10
